src/helpers/getColorPalette.ts

```diff
@@ -0,0 +1,9 @@
+import Vibrant from 'node-vibrant';
+import { mapValues } from 'lodash';
+import { ColorPalette } from '../typings/schema';
+
+export const getColorPalette = async (buffer: Buffer): Promise<ColorPalette> => {
+  const palette = await Vibrant.from(buffer).getPalette();
+
+  return mapValues(palette, color => (color ? color.getHex() : null));
+};
```

Comment:
Where is this file being used? Also, why is it part of the API? 🤔 I thought calculating the palette would be part of the job that retrieves the pictures?

Reply:
I think you're right.